config/secrets.yml

```diff
@@ -38,6 +38,7 @@ development:
   smtp_address: <%= ENV["MAILER_SMTP_ADDRESS"] %>
   smtp_domain: <%= ENV["MAILER_SMTP_DOMAIN"] %>
   smtp_port: <%= ENV["MAILER_SMTP_PORT"] %>
+  smtp_from: <%= ENV["MAILER_SMTP_FROM"] %>
```

Comment:
we should add this new env var to `config/initializers/figaro.rb`

Reply:
rigth!